openmls/src/group/core_group/new_from_external_init.rs

```diff
@@ -29,28 +25,29 @@ impl CoreGroup {
         backend: &impl OpenMlsCryptoProvider,
         params: CreateCommitParams,
         tree_option: Option<&[Option<Node>]>,
-        verifiable_public_group_state: VerifiablePublicGroupState,
+        group_info: GroupInfo,
+        external_pub: &[u8],
```

Comment:
Use `ExternalPub` extension instead.

Reply:
Right, it's an extension in the `GroupInfo`.